CHANGELOG.md

```diff
@@ -3,3 +3,10 @@
 ### Updated
 
 - Renamed SPM package name to `MangopayVaultSDK` from `MangopayVault`
+
+## [1.0.9] - 2024-05g-06
+
+### Updated
```

Comment:
@elviva-mgp  most recent change should come first at the top.

Reply:
Noted